### PR TITLE
Implement the `read_file` function

### PR DESCRIPTION
**Pull request by Moohan**
On hold until #584 is merged.

### CODE REVIEW

**Review by Jennit07 — CHANGES_REQUESTED** [review]
Not sure if something weird going on in posit, but some of the changes in here are not showing for me in R e.g. `process_extract_alarms_telecare` doesnt look like its changed for me but it has on the git changes. I've made sure i'm up to date with the branch etc.

Some of them have also been missed like `read_lookup_xxx`. I need to spend a bit more time looking through this to double check/push any changes but i'll leave this on request changes for now. If anyone else has time to look at this feel free to pick it up this week, otherwise i'll come back to it on monday

File: R/process_extract_alarms_telecare.R [hunk 1]
```diff
@@ -27,7 +27,7 @@ process_extract_alarms_telecare <- function(file_path = get_sc_at_episodes_path(
   client_table <- readr::read_rds(client_lookup_path)
```
Comment (Jennit07) [inline R/process_extract_alarms_telecare.R:27]:
```suggestion
  client_table <- read_file(client_lookup_path)
```

**Comment by Moohan** [comment]
> Hey @Moohan I've pushed some changes, can you please check you are happy with them? Then I'll approve/merge. Thanks

Looks good to me!

**Review by Jennit07 — APPROVED** [review]
Ready to merge

**Pull request merged**